pgjdbc/src/main/java/org/postgresql/ssl/PKCS12KeyManager.java

```diff
@@ -166,10 +166,10 @@ private synchronized void loadKeyStore() throws Exception {
           PSQLState.CONNECTION_FAILURE, ucex);
       } else {
         error =
-          new PSQLException(
-            GT.tr("Could not read password for SSL key file by callbackhandler {0}.",
-              cbh.getClass().getName()),
-            PSQLState.CONNECTION_FAILURE, ucex);
+            new PSQLException(
+              GT.tr("Could not read password for SSL key file by callbackhandler {0}.",
+                cbh.getClass().getName()),
+              PSQLState.CONNECTION_FAILURE, ucex);
```

Comment:
https://github.com/pgjdbc/pgjdbc/blob/ff6c8229282afd10599321a6055635306fe1c6fd/pgjdbc/src/main/checkstyle/checks.xml#L140-L147

LineWrappingIndentation gives the scale of indent space when the code is extended to a multiple lines.
According to rule the `lineWrappingIndentation` is 4. Earlier the new keyword was indented 2 spaces, since the `lineWrappingIndentation` is 4, there is a new error thrown by checkstyle.